Makefile

```diff
@@ -40,6 +40,7 @@ GOLANGCI_LINT = $(BINDIR)/golangci-lint
 # we keep it fixed to avoid it from unexpectedly failing on the project
 # in case of a version bump
 GOLANGCI_LINT_VER = v1.62.2
+GOPROXY ?=
```

Comment:
if left unset it will access the sources directly from what i understand.

the default is:
```
go env GOPROXY
https://proxy.golang.org,direct
```



https://go.dev/ref/mod#environment-variables

Reply:
We'll use what user provides or en empty value, which means 'direct'